deps/cares/src/ares_init.c

```diff
@@ -1748,7 +1767,14 @@ static int init_by_resolv_conf(ares_channel channel)
     /* Only update search domains if they're not already specified */
     update_domains = (channel->ndomains == -1);
 
-    fp = fopen(PATH_RESOLV_CONF, "r");
+    /* Support path for resolvconf filename set by ares_init_options */
+    if(channel->resolvconf_path) {
```

Comment:
Node's C++ style has a space here (yes, confusing since c-ares has the all-squished-up style), run `make lint-cpp` and you should seen an error